packages/nextclade-cli/src/io/http_client.rs

```diff
@@ -112,3 +125,26 @@ impl HttpClient {
     Ok(content)
   }
 }
+
+fn extra_ca_certs() -> Result<impl IntoIterator<Item = TrustAnchor<'static>>, Report> {
+  match env::var_os("NEXTCLADE_EXTRA_CA_CERTS") {
+    Some(filename) => {
+      let file = File::open(filename.clone())
+        .wrap_err_with(|| format!("When opening NEXTCLADE_EXTRA_CA_CERTS file {filename:?}"))?;
+
+      let mut reader = BufReader::new(file);
+
+      let certs = rustls_pemfile::certs(&mut reader)
+        .map(|c| c.wrap_err_with(|| "When parsing an extra CA certificate".to_owned()))
+        .collect::<Result<Vec<_>, Report>>()?;
+
+      let anchors = certs
+        .into_iter()
+        .map(|c| anchor_from_trusted_cert(&c).wrap_err_with(|| "When converting an extra CA certificate to a trust anchor".to_owned()).map(|a| a.to_owned()))
+        .collect::<Result<Vec<_>, Report>>()?;
+
+      Ok(anchors)
+    }
+    None => Ok(vec![])
+  }
+}
```

Comment:
@ivan-aksamentov I'd appreciate any feedback you have on this code. This isn't the first Rust I've written, but I'm still pretty new to it!

Reply:
Thanks!